### PR TITLE
New tileset

### PR DESCRIPTION
**Pull request by fourlastor**
PR updating the tileset:

https://user-images.githubusercontent.com/1263058/200172848-2bd1e8d7-55fa-420c-b867-b7eba9e40836.mp4

There's a weird bug with `YSort` when sorting things with Y < 0, not sure what's going on here:

image

### CODE REVIEW

File: project.godot [hunk 1]
```diff
@@ -74,4 +74,5 @@ common/enable_pause_aware_picking=true
 [rendering]
 
 quality/driver/driver_name="GLES2"
+2d/snapping/use_gpu_pixel_snap=true
```
Comment (fourlastor) [inline project.godot:77]: I keep seeing this changing - not sure which option should be correct

**Comment by fourlastor** [comment]
trigger format

File: project.godot (hunk 1 above)
Comment (takecare) [reply project.godot:77]: maybe i changed it? this is one of the things we need to decide on. do we want pixel snapping or not? idk why it's GPU but godot used to have different settings for pixel snapping

File: tilemap/Map.cs [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
+        {
+            hasKey = true;
+            walls.SetCellv(position, TileMap.InvalidCell);
+        }
     }
 
     public void PlayerAction(Vector2 direction)
     {
-        GD.Print("Player acting at " + player.Position + ", direction" + direction);
+        var position = PlayerMapPosition() + direction;
```
Comment (takecare) [inline tilemap/Map.cs]:
```suggestion
        var playerPosition = PlayerMapPosition() + direction;
```

File: tilemap/Map.cs [hunk 3]
```diff
@@ -1,18 +1,57 @@
 using Godot;
-using System;
+using Godot.Collections;
 
 public class Map : Node2D
 {
     private Player player;
+    private TileMap walls;
+    private bool hasKey;
```
Comment (takecare) [inline tilemap/Map.cs]:
```suggestion
    private bool playerHasKey;
```

File: tilemap/Map.cs (hunk 2 above)
Comment (fourlastor) [reply tilemap/Map.cs]: [1d63c9f](https://github.com/fourlastor-jams/game-off-2022/pull/12/commits/1d63c9f20905791991e6e18f2a3d906fe468a0fd)

**Review by takecare — APPROVED** [review]

**Pull request merged**